### ISSUE TITLE
clicking three dots on video embeds jumps to bottom of timeline and related bugs

### ISSUE THREAD

**Issue by Ethan-Sharp** [vector-im/element-web#20378]
### Steps to reproduce

occurs in desktop version

In any room, scroll up to a previously posted/uploaded embedded video file.
Click the 3 dots (vertical ellipses) at the bottom right of the video. (you don't have to start watching it first, but the bug still occurs if you do)
This will immediately scroll you down to the bottom of the message feed/timeline.

additional behavior of the ellipses:
If you play the video and then go into full screen, the 3 dots work correctly and a menu pops up. this changes the behavior of the bug after exiting full screen mode.
After clicking on and off of the 3 dots while in full screen mode and then exiting (as far as I know how to recreate this), if you press the 3 dots on the same video, sometimes the menu pops up half way down the screen from the embed instead of jumping.
I think whether it shows up or not when doing this depends on if the video is near the bottom already and where specifically you have scrolled.

mentioned and possible related/duplicate bugs I have found:
[19980](https://github.com/vector-im/element-web/issues/19980)
[20347](https://github.com/vector-im/element-web/issues/20347)

#### What did you expect?
For video options to show up
#### What happened instead?
teleports to the most recent message (bottom of feed)

### Operating system

Windows 10

### Application version

Element version: 1.9.8 Olm version: 3.2.8

### Homeserver

matrix.org

### Will you send logs?

No

**Comment by kittykat** [comment]
Can confirm also on Linux, flatpak install.

**Comment by t3chguy** [comment]
A jump also happens when clicking the mute/unmute button on a video in Firefox

**Comment by t3chguy** [comment]
Oh actually that is unrelated and happens due to the timestamp in bubble layout conflicting with the controls

**Comment by t3chguy** [comment]
https://user-images.githubusercontent.com/2403652/148404068-ff793f14-fda6-4b8f-b281-88efb613f453.mov


I unfortunately cannot reproduce this bug.

**Comment by t3chguy** [comment]
> occurs in desktop version

If this is only reproducible in the desktop app then I'm wagering it'll be an Electron/Chromium bug which updating Electron will resolve

**Comment by kittykat** [comment]
I'm on electron-16.0.4

**Comment by t3chguy** [comment]
Hmm, that's strange as Element Desktop is designed to run only with 13.5 at this time - https://github.com/vector-im/element-desktop/blob/develop/package.json#L64

**Comment by t3chguy** [comment]
> Can confirm also on Linux, flatpak install.

The flatpak is just a repackage of the `.deb` so are you sure that your Element Desktop is actually using 16.0.4 vs you just having 16.0.4 installed?

**Comment by t3chguy** [comment]
I can reproduce in Element Nightly (Electron 13.5) but not latest stable Chrome or FF, so presuming to be an Electron bug until more info is provided

**Comment by kittykat** [comment]
No, you're right - of course the flatpak isn't using my installed version so it will be an older one.

**Comment by dbkr** [comment]
re-opening because we downgraded electron again
